webgpu.h

```diff
@@ -67,6 +67,9 @@
 #else
 #  define _wgpu_MAKE_INIT_STRUCT(type, value) value
 #endif
+#define _wgpu_COMMA ,
+#define _wgpu_TRUE 1
+#define _wgpu_FALSE 0
```

Comment:
nit: I think I prefer to just inline the `0` so we don't have as many "pinky promise private" macros.

(same for `_wgpu_TRUE` but it's unused)